linux/ibus-keyman/tests/scripts/teardown-tests.sh

```diff
@@ -3,4 +3,10 @@ set -eu
 
 . "$(dirname "$0")/test-helper.inc.sh"
 
+if [ -v KEYMAN_PKG_BUILD ]; then
+  # Skip setup during package builds - can't run headless and we won't
+  # run the other tests anyway
+  exit 0
+fi
```

Comment:
Since this block is also in setup-tests.sh, would it be worth adding this to test-helper.inc.sh?

Reply:
done